### PR TITLE
Fix broken link for qrel.dev.small.tsv

### PR DESCRIPTION
**Pull request by larryli1999**
Update the link to download qrels.dev.small.tsv

### CODE REVIEW

File: docs/experiments-monot5-tpu.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Let's start.
 cd ${DATA_DIR}
 wget https://storage.googleapis.com/duobert_git/run.bm25.dev.small.tsv
 wget https://www.dropbox.com/s/hq6xjhswiz60siu/queries.dev.small.tsv
-wget https://www.dropbox.com/s/5t6e2225rt6ikym/qrels.dev.small.tsv
+wget https://www.dropbox.com/s/ie27l0mzcjb5fbc/qrels.dev.small.tsv?dl=1
```
Comment (rodrigonogueira4) [inline docs/experiments-monot5-tpu.md:74]: remove `?dl=1`?

**Review by rodrigonogueira4 — APPROVED** [review]
LGTM. Just a minor comment.

**Comment by lintool** [comment]
hey @larryli1999 @ronakice are these the same ones here? https://github.com/castorini/anserini/tree/master/src/main/resources/topics-and-qrels

If so, fetch from there. That's stable.

**Comment by larryli1999** [comment]
I have updated the github links as well as the corresponding file names in the documentation.

**Comment by ronakice** [comment]
Okay LGTM after the new edits, merging! Thanks :)

**Pull request merged**